types/react-native-video/index.d.ts

```diff
@@ -29,6 +29,11 @@ export interface OnProgressData {
   seekableDuration: number;
 }
 
+export interface OnSeekData {
+  currentTime: number;
+  seekTime: number;
```

Comment:
I'm not familiar with `react-native-video` but I couldn't find `seekTime` on https://github.com/react-native-community/react-native-video/search?q=seektime&unscoped_q=seektime.  Do you have an example usage?  (If so, it might make a good test.)